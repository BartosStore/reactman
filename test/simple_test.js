import React from 'react';
import GithubRepository from '../src/githubRepository';
import renderer from 'react-test-renderer';

describe('simple', () => {
  it('renders correctly', () => {
    const component = renderer
    .create(<GithubRepository name={'name'} description={'description'} language={'language'} key={'github-repo-0'} />)
    .toJSON();
    expect(component).toMatchSnapshot();
  });
});

describe('simple', () => {
  it('should be sane', () => {
    expect(false).not.toBe(true);
  });
});
