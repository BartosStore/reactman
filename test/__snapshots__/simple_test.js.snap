// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple renders correctly 1`] = `
<div>
  <h3>
    name
  </h3>
  <div>
    Description: 
    description
  </div>
  <div>
    Language: 
    language
  </div>
</div>
`;
